### PR TITLE
[MBL-17356][Student] - Fix front page title bug when navigating to front page from CourseBrowser 'Home'

### PR DESCRIPTION
**Pull request by kdeakinstructure**
Test plan: Test with a front page as the 'Home' of a course. Smoke test some other 'Home' settings and the Pages tab itself.

Fix front page issue by passing homePageTitle as the parameter for the TabHelper.getRouteByTabId method (null safely).

refs: MBL-17356
affects: Student
release note: Front Page title fixed when navigating from Course Browser by 'Home'.

## Screenshots

<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td></td>
<td></td>
</tr>
</table>

## Checklist

- [x] Follow-up e2e test ticket created or not needed
- [x] Tested in light mode

### CODE REVIEW

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

File: apps/student/src/main/java/com/instructure/student/util/TabHelper.kt [hunk 1]
```diff
@@ -71,7 +74,7 @@ object TabHelper {
 
     fun isHomeTab(tab: Tab): Boolean = "home".equals(tab.tabId, ignoreCase = true)
 
-    fun getRouteByTabId(tabb: Tab?, canvasContext: CanvasContext): Route? {
+    fun getRouteByTabId(tabb: Tab?, canvasContext: CanvasContext, homePageTitle: String? = null): Route? {
```
Comment (tamaskozmer) [inline apps/student/src/main/java/com/instructure/student/util/TabHelper.kt:77]: This parameter is unused

**Review by tamaskozmer — CHANGES_REQUESTED** [review]

File: apps/student/src/main/java/com/instructure/student/util/TabHelper.kt (hunk 1 above)
Comment (kdeakinstructure) [reply apps/student/src/main/java/com/instructure/student/util/TabHelper.kt]: Forgot to delete, thanks!

**Comment by inst-danger** [comment]
<p>Student Install Page</p>

**Review by tamaskozmer — APPROVED** [review]

**Review by kristofnemere — APPROVED** [review]
QA+1

**Pull request merged**